core/src/data/rasterSource.h

```diff
@@ -36,7 +36,7 @@ class RasterSource : public TileSource {
     // TODO Is this always PNG or can it also be JPEG?
     virtual const char* mimeType() override { return "image/png"; };
 
-    void loadTileData(std::shared_ptr<TileTask> _task, TileTaskCb _cb);
+    virtual void loadTileData(std::shared_ptr<TileTask> _task, TileTaskCb _cb);
 
```

Comment:
Removed the commit, and lets try to make it consistent as and when one touches that appropriate code.